api/views/CloudRecordView.py

```diff
@@ -113,6 +113,10 @@ def _signer_is_valid(self, signer_dn):
         signer_split = signer_dn.split("=")
         signer = signer_split[len(signer_split)-1]
 
+        if signer_dn in settings.BANNED_FROM_POST:
```

Comment:
`settings` only changes once the django is restarted (settings re-imported, and the file re-read), so you can't change the banned list dynamically.

The ban lists should be separated from the conf and re-read when checking AuthZ, or you should comment the conf and document that it requires a web server restart.